documentation/src/main/asciidoc/topics/integrations.adoc

```diff
@@ -2,16 +2,6 @@
 = Integrations
```

Comment:
this section looks weird now. We have:
```
1. Integrations
1.1 Apache Lucene
2. Hibernate
3. Spring
```

IMO, it would be better to "promote" Apache Lucene to top-level (so it would be 1 instead of 1.1) or bring the hibernate/spring one level down (1.2 hibernate, 1.3 spring)

WDYT?

Reply:
changed. thanks!